packages/source-airtable/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "@gridsome/source-airtable",
+  "version": "0.0.1",
+  "description": "Airtable source for Gridsome",
+  "homepage": "https://github.com/gridsome/gridsome/tree/master/packages/source-airtable",
+  "repository": "https://github.com/gridsome/gridsome/tree/master/packages/source-airtable#readme",
+  "main": "index.js",
+  "keywords": [
+    "gridsome",
+    "gridsome-plugin",
+    "gridsome-source",
+    "airtable"
+  ],
+  "dependencies": {
+    "airtable": "^0.6.0"
+  }
```

```suggestion
  },
  "engines": {
    "node": ">=8.3"
  }
```
This will probably make the linter pass.

---

Done.